app/app.go

```diff
@@ -72,14 +72,15 @@ import (
 	foundationclient "github.com/Finschia/finschia-sdk/x/foundation/client"
 	foundationkeeper "github.com/Finschia/finschia-sdk/x/foundation/keeper"
 	foundationmodule "github.com/Finschia/finschia-sdk/x/foundation/module"
-	"github.com/Finschia/finschia-sdk/x/genutil"
-	genutiltypes "github.com/Finschia/finschia-sdk/x/genutil/types"
 	"github.com/Finschia/finschia-sdk/x/gov"
 	govkeeper "github.com/Finschia/finschia-sdk/x/gov/keeper"
 	govtypes "github.com/Finschia/finschia-sdk/x/gov/types"
 	"github.com/Finschia/finschia-sdk/x/mint"
 	mintkeeper "github.com/Finschia/finschia-sdk/x/mint/keeper"
 	minttypes "github.com/Finschia/finschia-sdk/x/mint/types"
+	"github.com/Finschia/finschia-sdk/x/or/rollup"
+	rollupkeeper "github.com/Finschia/finschia-sdk/x/or/rollup/keeper"
```

Comment:
no need rollup module import ?

Reply:
The import of `rollup` module is required for `rollup.AppModuleBasic{}` at line 148.

Reply:
It is not used in simd of finschia-sdk (for-ramus-sdk branch), but is it necessary for finschia?
https://github.com/Finschia/finschia-sdk/blob/for-ramus-sdk/simapp/app.go#L114

Reply:
Rollup modules is not needed in L2 application.

Reply:
That should be added in sims. I think it just looks fine because `AppModuleBasic` in `x/or/rollup` now has very little valid behaviour. It'll be needed when we have an API for Rollup, for example, or when we start using the default genesis.

Reply:
All usage points for rollup-related modules included in L2 have been removed since rollup isn't used in L2.